modules/about-base-training-images.adoc

```diff
@@ -0,0 +1,64 @@
+:_module-type: CONCEPT
+
+[id="about-base-training-images_{context}"]
+= About base training images
+
+[role="_abstract"]
+The base training images for distributed workloads are optimized with the tools and libraries that you need to run distributed training jobs. 
+You can use the provided base images, or an {productname-short} admin user can create custom images adapted to your needs.
+
+ifndef::upstream[]
+For information about Red Hat support of training images and packages, see link:https://access.redhat.com/articles/rhoai-supported-configs[{productname-long}: Supported Configurations].
+endif::[]
+
+The following table lists the training images that are installed with {productname-long} by default.
+
+If the preinstalled packages that are provided in these images are not sufficient for your use case, you have the following options:
+
+* Install additional libraries after launching a default image. 
+This option is good if you want to add libraries on an ad·hoc basis as you run training jobs. 
+However, it can be challenging to manage the dependencies of installed libraries, and your changes are not saved when the workbench restarts.
```

Comment:
This phrase can be removed as it's only for workbench.

Reply:
I removed the second half of the third sentence, because I _think_ the rest of the text applies to training images. Please let me know if I should remove more text.